doc/dev/deduplication.rst

```diff
@@ -149,3 +149,64 @@ Aside from radosgw, completing work on manifest object support in the
 OSD particularly as it relates to snapshots would be the next step for
 rbd and cephfs workloads.
 
+How to use deduplication
+========================
+
+Currently, Ceph provides deduplication using RADOS machinery.
+The below is explanation about how users use deduplication.
+
+
+1. Find dedupable chunks using ceph-dedup-tool. ::
```

Comment:
The output of this operation prints how much the data in a target pool can be saved if dedup is operated. 
So, how about "Estimate space saving ratio of a target pool"?